lib/add_magic_comment.rb

```diff
@@ -3,14 +3,15 @@
 # A simple library to prepend magic comments to multiple ".rb" files
 
 module AddMagicComment
-  MAGIC_COMMENT_PREFIX  = "frozen_string_literal"
-  MAGIC_COMMENT_PATTERN = /^-?# *#{MAGIC_COMMENT_PREFIX}/
+  MAGIC_COMMENT_PREFIX  = 'frozen_string_literal'
```

Comment:
Yeah, as I said, it's largely a matter of taste.  I actually also like knowing that no special characters occur in a string, though that is of way less importance than knowing that there's no interpolation going on inside it.  But it's fine, I'll revert to double quotes.